BUILD.gn

```diff
@@ -250,6 +257,8 @@ if (current_toolchain != "${dir_pw_toolchain}/default:default") {
 
     # Build the k32w shell app example.
     enable_k32w_shell_app_build = enable_k32w_builds
+
+    enable_fake_tests = enable_default_builds && host_os == "linux"
```

Comment:
What's wrong with these tests on mac?

Reply:
I have no way to test this on mac, so I restricted this to what was tested.